### ISSUE TITLE
Design Guideline /Best Practises for new icons is missing

### ISSUE THREAD

**Issue by suterma** [Templarian/MaterialDesign#6879]
### What new documentation or how-to guide would you like to see?

This seems quite basic, and I apologize if it's obvious, but I fail to find a document that helps me to design new icons with the "Material Design" style.

I read [about contributing](https://github.com/Templarian/MaterialDesign/blob/0e37c439b52a98b2704f705f97711ba3a0a4399e/.github/CONTRIBUTING.md)
and looked over at Google, but the [Material Icons Guide](https://developers.google.com/fonts/docs/material_icons) did not help me with designs.

Maybe there is some "Template Icon" with some available features in the the thickness, radius etc. that is some kind of standard here?

**Comment by mririgoyen** [comment]
This is what you want to read through: https://material.io/design/iconography/system-icons.html#design-principles

I agree our documentation isn't the best around this. Trust me, I'm actually working on it right now. Our new site is in the works, and it will contain better and more up-to-date documentation.

**Comment by suterma** [comment]
Yes, this is exactly what I was looking for! I suggest you place this link somewhere where it's about contributing.
